config.default

```diff
@@ -286,11 +286,7 @@ movingAveragePoints = 12
 # compare to observations?
 compareWithObservations = True
 # list of region indices to plot from the region list in [regions] below
-regionIndicesToPlot = [6]
-# Number of points over which to compute moving average (e.g., for monthly
-# output, movingAveragePoints=12 corresponds to a 12-month moving average
-# window)
-movingAveragePoints = 12
+regionIndex = 5
```

Comment:
@xylar, @vanroekel, about regions, region names/indexes: at some point, we will have to update a few AM's (including the `surfaceAverages` used here, and `layerWeightedAverages` used for the OHC plot) to include better handling of ocean basin and regional calculations, using maskfiles instead of the hardwired regions that we have now. When that time comes (right now this is not a huge priority), we can get regionNames and regionGroupNames from the mask files, or maybe it could be part of the AM output.
About removing `compareWithObservations`: we probably need to do a bit of cleanup in general, although not in this PR, also including `compareWithModel`.